node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -269,6 +269,22 @@ class NodeRuntime private[node] (
   private def rpConf(local: PeerNode, bootstrapNode: Option[PeerNode]) =
     RPConf(local, bootstrapNode, defaultTimeout, rpClearConnConf)
 
+  implicit def effectParallel = new Parallel[Effect, EffectPar] {
+    override def applicative: Applicative[EffectPar] = Applicative[EffectPar]
+    override def monad: Monad[Effect]                = Monad[Effect]
+    override def sequential: ~>[EffectPar,Effect] = new ~>[EffectPar, Effect] {
+      override def apply[A](par: EffectPar[A]): Effect[A] =
+        EitherT.liftF[Task, CommError, A](Task.Par.unwrap(par.value).flatMap {
+        case Left(ex) => Task.raiseError(new Exception(ex.toString))
+        case Right(x) => Task.pure(x)
+      })
+    }
+
+    override def parallel: ~>[Effect, EffectPar]     = new ~>[Effect, EffectPar] {
+      override def apply[A](eff: Effect[A]): EffectPar[A] = ??? //TODO
+    }
+  }
```

Comment:
@rabbitonweb @ArturGajowy Can you double check this implementation?

Reply:
this is incomplete. I forgot to put the label WIP to this PR

Reply:
so I removed this implementation and I used Parallel.identity instead. Thanks @ArturGajowy for the pointer